dashboard/test/mailers/previews/pd_teacher_application_mailer_preview.rb

```diff
@@ -2,28 +2,46 @@
 class Pd::TeacherApplicationMailerPreview < ActionMailer::Preview
```

Comment:
This is replacing the mailer preview for the 17-18 version of the teacher application. (Which will be deleted in this PR: https://github.com/code-dot-org/code-dot-org/pull/30805)